README.md

```diff
@@ -43,8 +43,31 @@ The Azure AI Content Safety API uses severity levels from 0-6:
 
 This repository is licensed under the [Mattermost Source Available License](LICENSE) license.
 
+## Frequently Asked Questions
+
+### How does content moderation work?
+
+When a user posts a message, it appears immediately in the channel. The plugin then analyzes the content in the background using Azure AI Content Safety APIs. If harmful content is detected, the post is automatically deleted and notifications are sent to inform users of the removal.
+
+### Will I still receive notifications for harmful content?
+
+Currently, yes. Push notifications may be sent for posts that contain harmful content before the moderation process completes. This is because notifications are typically sent immediately when posts are created, while content analysis happens asynchronously. We are working to improve this behavior (see roadmap).
+
+### Can I exclude certain users from moderation?
+
+Yes, you can specify user IDs in the "Excluded Users" configuration setting. All other users will have their content moderated automatically.
+
+### What if content moderation APIs are unavailable?
+
+The plugin uses a "fail-open" approach for reliability. If the moderation API is unavailable or returns an error, no posts are moderated.
```

Comment:
Is there something specific in logs that can be used to monitor for unavailability?

Reply:
> Yes there is - they'll see a log statement with this error: `"Content moderation error: moderation service is not available"`

Great. Please add that to the faq.